_stbt/types.py

```diff
@@ -272,9 +272,18 @@ def to_slice(self):
                       max(0, self.right)))
 
     def contains(self, other):
-        """:returns: True if ``other`` is entirely contained within self."""
-        return (other and self.x <= other.x and self.y <= other.y and
-                self.right >= other.right and self.bottom >= other.bottom)
+        """:returns: True if ``other`` (a `Region` or `Position`) is entirely
+        contained within self.
+        """
+        if all(hasattr(other, a) for a in ("x", "y", "right", "bottom")):
+            # a Region
+            return (self.x <= other.x and self.y <= other.y and
+                    self.right >= other.right and self.bottom >= other.bottom)
+        elif hasattr(other, "x") and hasattr(other, "y"):  # a Position
+            return (self.x <= other.x <= self.right and
+                    self.y <= other.y <= self.bottom)
+        else:
+            return False
```

Comment:
This should probably raise instead of returning False.

Reply:
Done. It still returns False if other=None. Unlike the previous implementation it checks explicitly for `None` instead of `not other`, because numpy arrays would raise a confusing error message.